### PR TITLE
Migrate Response.js and Responses.js to styled components

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/3289

# Screenshots

<details>
<summary>Desktop</summary>

image
</details>

<details>
<summary>Mobile</summary>

Screen Shot 2020-11-06 at 13 06 14
</details>

### CODE REVIEW

**Comment by SudharakaP** [comment]
@Betree : Can you tell me in which page these components are used please? Have trouble finding them. 😺

**Comment by SudharakaP** [comment]
> @Betree : Can you tell me in which page these components are used please? Have trouble finding them. smiley_cat

@Betree : No worries. Found the place. I've updated the screenshots section. 👍🏽

File: components/Response.js [hunk 1]
```diff
@@ -47,51 +57,26 @@ class Response extends React.Component {
...
+            alignItems="flex-start"
+            width="100%"
+            margin="10px"
+            maxWidth="300px"
+            float="left"
+            position="relative"
+            height="90px"
+            overflow="hidden"
+          >
+            {status === 'INTERESTED' && <StarObject title={title} type="image/svg+xml" data={star} className="star" />}
```
Comment (Betree) [inline components/Response.js:71]: From what I see in https://github.com/opencollective/opencollective-frontend/blob/a4bb9656cfbaccbb224922cdb6c5e91b39c52976/components/collective-page/sections/SponsorsAndParticipants.js#L61, it seems that this "INTERESTED" status is not used anymore. Let's take the opportunity to kill it :smiling_imp:

Comment (SudharakaP) [reply components/Response.js]: Done 😄

**Review by Betree — APPROVED** [review]
Nice! Great to see this old code going away :smiley:

**Pull request merged**